portfolio/src/main/webapp/index.html

```diff
@@ -69,9 +69,45 @@ <h2>Image Gallery</h2>
         <h2>Projects</h2>
         <ul>
             <li> Web Photosharing Application </li>
+            <h4> Features</h4>
+            <ul>
+                <li> Register new users and login page</li>
+                <li> Upload photos and manage photo visibility</li>
+                <li> Comment under users'photos</li>
+            </ul>
+            <h4> Technical Stack</h4>
+            <ul>
+                <li> HTML</li>
+                <li> Javascript</li>
+                <li> CSS </li>
+                <li> NodeJS </li>
+                <li> ReactJS </li>
+            </ul>
+            <br></br>
             <li> Website customization to send cutting and edging measurements to backend of Shopify</li>
-            <li> Spotfire Visual representation of oil sites based on the location of ONEOK's contracted suppliers in the Oklahoma and Kansas region </li>
+            <h4> Features</h4>
+            <ul>
+                <li> User enters the dimensions of the cuts from a desired board</li>
+                <li> Once a form is submitted, sends a request to the admin portal of Shopify, and identifies the form submission as a new order</li>
+            </ul>
+            <h4> Technical Stack</h4>
+            <ul>
+                <li> HTML</li>
+                <li> Javascript</li>
+                <li> CSS </li>
+                <li> NodeJS </li>
+            </ul>
+            <br></br>
             <li> Security application to monitor ONEOK's wifi access points </li>
+            <h4> Features</h4>
+            <ul>
+                <li> Routinely sends list of detectable wifi access point names to database </li>
+            </ul>
+            <h4> Technical Stack</h4>
+            <ul>
+                <li> Python</li>
+                <li> Raspberry Pi</li>
+            </ul>
         </ul>
         <h2>Contact me!</h2>
          <a target="_blank" href="https://www.linkedin.com/in/david-ihim-404750143"> LinkedIn </a>
```

Comment:
nit: please align with other children of the same depth (

Reply:
A bit confused on this one. Is the <ul> tag not inline with the <h4> tag above?

Reply:
Oh I just meant the `<a>` has some extra spacing compared to the `<h2>`

Reply:
Oh ok gotcha, fixed